docs/manual/docs/user-guide/workflow/life-cycle.md

```diff
@@ -1,37 +1,41 @@
 # Life cycle
 
-## Record life cycle
+Metadata records can have a lifecycle that typically goes through one or more states. This is an optional feature that 
+can be activated on demand (see [Activate the metadata workflow](#activate-workflow)).
 
-Metadata records have a lifecycle that typically goes through one or more states. For example, when a record is:
+For example, when a record is:
 
 -   created and edited by an `Editor` it is in the `Draft` state.
--   being reviewed by a `content reviewer`, or a review is requested brings the record to `Submitted` state.
+-   being reviewed by a `Content reviewer`, or a review is requested brings the record to `Submitted` state.
 -   completed and corrected by the `Content Reviewer` it is in the `Approved` state.
 -   superseded or replaced the state is `Retired`.
 
-The catalog has (an extensible) set of states that a metadata record can have:
+The catalog has a set of states that a metadata record can have:
 
--   `Unknown` - this is the default state - nothing is known about the status of the metadata record.
 -   `Draft` - the record is under construction or being edited.
 -   `Submitted` - the record has been submitted for approval to a content review.
 -   `Approved` - the content reviewer has reviewed and approved the metadata record.
--   `Rejected` - the content reviewer has reviewed and rejected the metadata record.
 -   `Retired` - the record has been retired.
 
-Workflow can be enabled for the full catalogue, certain groups or on an individual record level.
+When the metadata workflow is activated, the existing records are set in a special status `Unknown`.
 
-In the last case, to enable workflow and change the status from `Unknown` to `Draft`, click the `enable workflow` button in the metadata view:
+## Activate the metadata workflow {#activate-workflow}
+
+To enable the record life cycle, the metadata workflow should be activated. It can be activated for the full catalogue, certain groups or on an individual record level.
```

Comment:
```suggestion
To enable the record life cycle, activate the metadata workflow. It can be activated for the full catalogue, certain groups, or on an individual record.
```

Removing the word "should" in order to be direct.